docs/overview/component/gateway.md

```diff
@@ -0,0 +1,49 @@
+# Vald Gateway
+
+Vald Gateway is the component to handle all requests in the Vald cluster.
+
+This component is essential to operate the Vald cluster because all requests should pass it, and other components depend on it.
+
+This page introduces the overview and features of Vald Gateway.
+
+## Responsibility
+
+Vald Gateway is responsible for:
+
+- Pass requests to the other Vald components
+- Control request timeout
+- Sort out search results as user demand
+
+## Features
+
+Vald Gateway has two kinds of components:
+
+1. Vald LB Gateway
+   - The main component of the Vald cluster and connecting to Kubernetes Ingress.
```

Comment:
```suggestion
   - The main component of the Vald cluster and connecting to Kubernetes Ingress
```
I read this sentence as a noun phrase, so I think that `.` is not necessary because of aligning the expression as in other sentences.
( I have the idea of using the same expression as in the table. )